UefiTestingPkg/AuditTests/DMAProtectionAudit/UEFI/IVRS/IVRSAcpiTable.c

```diff
@@ -133,7 +133,11 @@ GetIvrsAcpiTableIvmd (
       case IVMD_TYPE_21H:
       case IVMD_TYPE_22H:
         // If IVMD found add to end of linked list
-        NewNode       = (IVMDListNode *)AllocateZeroPool (sizeof (IVMDListNode));
+        NewNode = (IVMDListNode *)AllocateZeroPool (sizeof (IVMDListNode));

```

Comment:
This could leave partially allocated list nodes if some were allocated in the loop before an allocation failure occurs.

Given our boot environment this isn't too bad, but it would be cleaner to walk the list and free nodes.